src/StarterGui/WesTimingGui/TimingLocalScript/TimingKeeperService.lua

```diff
@@ -40,7 +40,35 @@ function service:HandleSector(sector: number)
     elseif sector == 2 then
         sectorTime = time() - sector1At
         sector2At = time()
-    end
+	end
+	
+	-- Get delta time data
+	if _config.SectorTimeDelta == true then
+		
+		local deltaTimeData = _dataService:GetDeltaTime(sector, sectorTime, sectorIsValid)
+
+		-- Display delta time
+		if deltaTimeData ~= nil then
+
+			-- Popup data
+			local frameColor = _helpers:GetStatePopupColor(deltaTimeData.state)
+			local displayString = _helpers:ConvertTime(math.abs(deltaTimeData.delta))
+
+			-- Improved sector
+			if deltaTimeData.delta <= 0 then
+				displayString = "- " .. displayString
```

Comment:
I prefer no whitespace between `-` and the digits
```suggestion
				displayString = "-" .. displayString
```